templates/js/custom-copilot-rag-custom-api/README.md.tpl

```diff
@@ -1,20 +1,19 @@
-# Overview of the Basic AI Chatbot template
+# Overview of the Custom Coilot from Custom API template
 
-This template showcases a bot app that responds to user questions like ChatGPT. This enables your users to talk with the AI bot in Teams.
+This template showcases a bot app that calls real API according to OpenAPI spec, and response with adaptive card.
 
```

Comment:
This template showcases an AI-powered intelligent chatbot that can understand natural language to invoke the API defined in the OpenAPI description document.